src/GuppyDataExplorer/ExplorerTable/ExplorerTable.css

```diff
@@ -79,3 +74,17 @@
   color: #3283c8;
   fill: #3283c8;
 }
+
+.explorer-nested-table {
+  padding: 10px;
+  width: 75vw;
+}
+
+.explorer-nested-table .ReactTable .pagination-bottom .-pagination .-center {
+  display: none;
+  opacity: 0;
+  width: 0;
+  height: 0;
+  position: absolute;
+  border: solid red 1px;
```

Comment:
You can delete this line ~ I think I might even have written that line 😅

Reply:
oh right! i think i might actually only need the `display: none;` in here

will do